cl/corpus_importer/management/commands/update_opinions_order.py

```diff
@@ -0,0 +1,415 @@
+                if op.get("opinion")
+            ]
+            cl_opinions_content = [
+                op.get("opinion")
+                for op in cl_cleaned_opinions
+                if op.get("opinion")
+            ]
+
+            if len(columbia_opinions_content) != len(cl_opinions_content):
+                logger.warning(
+                    f"The number of opinions in cl and the number of opinions in the xml is different, cluster id: {cluster_id}"
+                )
+                continue
+
+            # Try to match content between cl and xml
+            matches = match_opinion_lists(
+                columbia_opinions_content,
+                cl_opinions_content,
+            )
+
+            if matches:
+                if len(matches.values()) != len(set(matches.values())):
+                    # We don't have a unique match for each opinion, they were
+                    # probably combined incorrectly
+                    logger.info(
+                        f"We can't infer opinions order for cluster id: {cluster_id}"
+                    )
+                    # Go to next cluster id
+                    continue
+
+                if len(cl_cleaned_opinions) > len(set(matches.values())):
+                    # We have more opinions than matches
+                    logger.info(
+                        f"We couldn't match all cl opinions to the file's "
+                        f"content, cluster id: {cluster_id}"
+                    )
+                    # Go to next cluster id
+                    continue
+
+                # All opinions matched, update all opinions order
+                update_opinions(
+                    cluster_id,
+                    cl_cleaned_opinions,
+                    extracted_columbia_opinions,
+                    matches,
+                )
+
+
+class Command(BaseCommand):
+    help = "Fill order field in Opinion objects"
+
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--process-harvard",
+            action="store_true",
+            help="Fix harvard opinions order",
+        )
+        parser.add_argument(
+            "--process-columbia",
+            action="store_true",
+            help="Fix columbia opinions order",
+        )
+        parser.add_argument(
+            "--xml-dir",
+            default="/opt/courtlistener/_columbia",
+            required=False,
+            help="The absolute path to the directory with columbia xml files",
+        )
+        parser.add_argument(
+            "--start-id",
+            type=int,
+            default=0,
+            help="Start id for a range of clusters (inclusive)",
+        )
+        parser.add_argument(
+            "--end-id",
+            type=int,
+            default=0,
+            help="End id for a range of clusters (inclusive)",
+        )
```

Comment:
I think we should revamp this.  I find the start and end id stuff kinda confusing and I dont know why we need it.  

lets switch this to skip-until and --limit 

Otherwise I dont know how you would know what the end id you want to go until is.